### PR TITLE
More Fixes to Crypto + key sort

### LINKED ISSUE #3017: [Bug] `crypto/find <coin> --source` warnings and errors
**Describe the bug**
Source flag
`find eth --source CoinPaprika` -> FutureWarning, No data for second table
`find eth --source Binance` -> FutureWarning, No data for first table
`find eth --source Coinbase` -> FutureWarning, No data for both tables
`find eth --source Yahoofinance` -> Error: object of type 'float' has no len()

**To Reproduce**
`crypto/find eth --source <choices>`

**Screenshots**
Example of FutureWarning and no data for Coinbase
Screen Shot 2022-10-24 at 10 54 40 AM

**Screenshots**
Please update `openbb_terminal/miscellaneous/scripts/cryptocurrency/test_crypto.openbb` by removing commented out commands.

### LINKED ISSUE #3048: [Bug] `crypto/ov/exchanges` bugs
**Describe the bug**

1. CoinPaprika source does not work
Screen Shot 2022-10-25 at 4 36 47 PM

2. Sort does not work on any column
Screen Shot 2022-10-25 at 4 39 25 PM

3. Descend is false, using the `--descend` flag does not make sense when it is ascending
Screen Shot 2022-10-25 at 4 40 26 PM
3. `--vs` flag does nothing.
Screen Shot 2022-10-25 at 4 44 49 PM

**To Reproduce**
`crypto/ov/exchanges`

### LINKED ISSUE #3056: [Bug] `crypto/ov/markets --sort` does not work
**Describe the bug**
1. The `--sort` flag does not work 
Screen Shot 2022-10-25 at 8 47 18 PM

**To Reproduce**
`crypto/ov/markets --sort`

### LINKED ISSUE #3095: [Bug] `crypto/onchain/baas` command is broken
**Describe the bug**
1. `baas` command does not work or display anything
Screen Shot 2022-10-26 at 3 06 36 PM

**To Reproduce**
crypto/onchain/baas -c <coin>

**Additional context**
After fixing, please ensure that the other optional flags are working properly and to additionally update `openbb_terminal/miscellaneous/scripts/cryptocurrency/test_crypto_onchain.openbb`

### PR DESCRIPTION
**Pull request by andrewkenreich**
# Description

- Improvement - Sorts keys by alphabetical order 

and fixes below bugs

- fix #3095 
- fix #3056 
- fix #3048 
- fix #3017 

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/cryptocurrency/overview/coinpaprika_model.py [hunk 1]
```diff
@@ -119,6 +119,7 @@ def get_list_of_coins() -> pd.DataFrame:
     coins = session.make_request(session.ENDPOINTS["coins"])
     df = pd.DataFrame(coins)
     df = df[df["is_active"]]
+    print(df)
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/overview/coinpaprika_model.py:122]: this intentional?

File: openbb_terminal/cryptocurrency/overview/coinpaprika_model.py [hunk 2]
```diff
@@ -272,7 +273,10 @@ def get_coins_market_info(
         "pct_from_ath",
     ]
     df = _get_coins_info_helper(symbols=symbols)[cols].sort_values(by="rank")
-    df.sort_values(by=sortby, ascending=ascend)
+    if sortby == "rank":
+        df = df.sort_values(by=sortby, ascending=False if ascend else True)
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/overview/coinpaprika_model.py:277]: `not ascend` ?

File: openbb_terminal/cryptocurrency/overview/overview_controller.py [hunk 3]
```diff
@@ -142,9 +146,7 @@ def __init__(self, queue: List[str] = None):
                 "--pie": {},
             }
             choices["exchanges"] = {
-                "--sort": {c: {} for c in pycoingecko_model.EXCHANGES_FILTERS}
-                if get_ordered_list_sources(f"{self.PATH}exchanges")[0] == "CoinGecko"
-                else {c: {} for c in coinpaprika_model.EXCHANGES_FILTERS},
+                "--sort": {c: {} for c in self.combined_filters},
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/overview/overview_controller.py:149]: Do we have --sort or --sortby throughout?

File: openbb_terminal/keys_model.py [hunk 4]
```diff
@@ -82,6 +82,9 @@
     "stocksera": "STOCKSERA",
 }
 
+# sorting api key section by name
+API_DICT = dict(sorted(API_DICT.items()))
```
Comment (jmaslek) [inline openbb_terminal/keys_model.py:86]: big brain

File: openbb_terminal/cryptocurrency/overview/coinpaprika_model.py (hunk 1 above)
Comment (andrewkenreich) [reply openbb_terminal/cryptocurrency/overview/coinpaprika_model.py:122]: Noooo needs to go

File: openbb_terminal/cryptocurrency/overview/coinpaprika_model.py (hunk 2 above)
Comment (andrewkenreich) [reply openbb_terminal/cryptocurrency/overview/coinpaprika_model.py:277]: Behind ascend

**Review by jmaslek — APPROVED** [review]

**Pull request merged**